lametro/wagtail_hooks.py

```diff
@@ -94,6 +92,19 @@ class Meta:
         fields = ["current_member"]
 
 
+class EventNoticeFilterSet(django_filters.FilterSet):
+    broadcast_conditions = django_filters.ChoiceFilter(
+        lookup_expr="icontains", choices=EventNotice.BROADCAST_CONDITION_CHOICES
+    )
+    comment_conditions = django_filters.ChoiceFilter(
+        lookup_expr="icontains", choices=EventNotice.COMMENT_CONDITION_CHOICES
+    )
```

Comment:
The filters on the listing page are using the easier to read (but often lengthy) version of the condition choices, while the list itself is using the shorter, less clear version.

It's probably best to use just one of those across both, but I wanted to check which you thought was the better format. I'm leaning towards to more clear one.

Reply:
Yeah, I think going with the longer version is worth it for the clarity.